aries-site/src/examples/components/pageheader/PageHeaderAnatomy.js

```diff
@@ -53,7 +53,7 @@ export const PageHeaderAnatomy = ({ background }) => {
     <Stack>
       <Grid
         align="center"
-        columns={['auto', 'flex', 'auto']}
+        columns={['30px', 'flex', '30px']}
```

Comment:
why fixed pixel values?

Reply:
matches the values for what was given for `rows` 
going to try another approach though